Gemfile

```diff
@@ -8,7 +8,7 @@ if branch = ENV['TRAVIS_BRANCH']
   same_branch_exists = `curl --head https://github.com/neo4jrb/neo4j-core/tree/#{branch} | head -1`.match(/200 OK/)
   gem 'neo4j-core', github: 'neo4jrb/neo4j-core', branch: same_branch_exists ? branch : 'master'
 else
-  gem 'neo4j-core', github: 'neo4jrb/neo4j-core'
```

Comment:
Did you mean to leave this in?

This was intended to serve as the option when there isn't a branch with the same name in the `neo4j-core` repository.  I'm definitely often specifying the local `../neo4j-core` path so maybe this whole if/else statement can exist inside of a `if ENV['CI']` and the `else` for that would be to use the local repo